config/module.config.php

```diff
@@ -31,6 +31,11 @@
             'AssetManager\Resolver\AliasPathStackResolver'      => 1000,
             'AssetManager\Resolver\PathStackResolver'           => 500,
         ),
+        'view_helper' => array(
+            'cache'            => null,
+            'query_string'     => '_',
+            'development_mode' => false,
```

Comment:
I'm not convinced about this key. Isn't there some sort of "development" flag already?